### PR TITLE
[13258] Update repos versions

### PR DESCRIPTION
**Pull request by JLBuenoLopez-eProsima**
Signed-off-by: JLBuenoLopez-eProsima <joseluisbueno@eprosima.com>

### CODE REVIEW

File: fastdds_statistics_backend.repos [hunk 1]
```diff
@@ -2,25 +2,25 @@ repositories:
...
         url: https://github.com/eProsima/Fast-DDS.git
-        version: master
+        version: v2.5.0
     fastdds_statistics_backend:
         type: git
         url: https://github.com/eProsima/Fast-DDS-statistics-backend.git
-        version: main
-    fastrtpsgen:
+        version: v0.4.0
+    fastddsgen:
```
Comment (rsanchez15) [inline fastdds_statistics_backend.repos:18]: This should be in another PR to be merged in main branch

**Review by rsanchez15 — CHANGES_REQUESTED** [review]

File: fastdds_statistics_backend.repos (hunk 1 above)
Comment (JLBuenoLopez-eProsima) [reply fastdds_statistics_backend.repos:18]: I have done it in master in this other one: #142

Comment (rsanchez15) [reply fastdds_statistics_backend.repos:18]: Then this is just a remainder:  `0.4.0` branch should be rebased to `main` and then `feature/update_repos` to `0.4.0`

Comment (JLBuenoLopez-eProsima) [reply fastdds_statistics_backend.repos:18]: Yes! Once all `0.4.0` milestone PRs have been merged it is going to be done.
